benchmarks/query/bin/runQueryTiming.sh

```diff
@@ -0,0 +1,8 @@
+# Run the build (after purging .cache) and output the amount of time
+# taken by the query execution phase
+#
+# run with `bin/runQueryTiming.sh`
+
+output=$(rm -rf .cache && gatsby build | grep "run graphql queries")
+echo $output | cut -d' ' -f 6
+
```

Comment:
this (kind of) limits where we can use this benchmark (cough - windows) :) but that's just utility to grab time for running queries, so I wouldn't say that it is a blocker

Reply:
It's a sign of the times when windows doesn't even enter my head. I'll leave it in as an optional useful tool for non windows users.